src/com/gfredericks/like_format_but_with_named_args.clj

```diff
@@ -25,9 +25,9 @@
 (defn throw-on-bad-format
   [format-string arg-map e]
 
-  (throw (ex-info "Unable to use named-format!" {:format-string format-string
-                                                 :arg-map       arg-map
-                                                 :cause         (.getMessage e)})))
+  (throw (ex-info "Bad call to named-format" {:format-string format-string
+                                              :arg-map    arg-map}
+                  (.getMessage e))))
```

Comment:
this should be just `e`, not `(.getMessage e)` -- the jvm `Throwable` class has a mechanism for attaching one exception to another (see [here](https://docs.oracle.com/javase/8/docs/api/java/lang/Throwable.html#getCause--)); in fact, I think the code as you have it will break, so I'm interested in whether there's a test that is failing.

Reply:
Woof, sorry. That was sloppy of me.